lib/Rex/Group/Lookup/XML.pm

```diff
@@ -33,7 +33,7 @@ use Rex -base;
 require Exporter;
 use base qw(Exporter);
 use vars qw(@EXPORT);
-XML::LibXML->require;
+use XML::LibXML;
```

Comment:
I don't think that we want to support until the end of extended lifecycle which is often only available with payment and so we also can't test on those distributions. Also (in regard of the mentioned ticket) CentOS 4 isn't really on the supported os list. On the other hand, people can also continue using an old version of rex (we don't delete the source and rpm/deb files from rex.linux-files.org) and so the people can also build a version on their own (or with support by us via irc) which can remove those dependencies they don't need.